CODE_OF_CONDUCT.md

```diff
@@ -6,6 +6,24 @@ The latest version of the upstream CNCF Code of Conduct is at https://github.com
 
 If the version reproduced below is out of date, the Code of Conduct which applies to the cert-manager project is the latest version upstream.
 
+## cert-manager Project Code of Conduct Committee
+
+Given the size of the team involved with cert-manager, there is no explicitly
+defined separate team of people responsible for policing the code of conduct
+for matters which fall exclusively under the [jurisdiction](https://github.com/cncf/foundation/blob/main/code-of-conduct/coc-committee-jurisdiction-policy.md) of the cert-manager project.
+
+If a committee is needed to evaluate a potential violation of the Code of Conduct,
+that team shall be made up of both the steering committee and maintainers. All
+steering committee members and all maintainers are invited, and all are expected to
+attend if they're able to.
+
+Any decisions made by the Code of Conduct committee should seek unanimous consent,
+but failing that should fall back to a majority vote.
+
+If the alleged violator is in the steering committee or the maintainer group,
```

Comment:
Does the CNCF provide any other level of CoCC that we could refer these matters to? I'm conscious if say, a contributor/non-maintainer has an issue with a maintainer or steering committee members behaviour, they may be discouraged from making the initial report in the knowledge that the alleged perpetrator would be involved in/aware of the report.

Reply:
Fair enough yeah; no reason not to be explicit. I've added a link to the relevant section and more prose.

Reply:
Those changes look great - way clearer and acknowledges up front that there may be some cases where people feel uncomfortable to report to all maintainers at once. Thank you! 🙌